autoscale-scaler-kubernetes/src/main/java/com/github/autoscaler/scaler/kubernetes/K8sHealthCheck.java

```diff
@@ -28,20 +29,22 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.List;
+
 final class K8sHealthCheck
 {
-    private static final Logger LOG = LoggerFactory.getLogger(K8sHealthCheck.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(K8sHealthCheck.class);
 
     private K8sHealthCheck()
     {
 
     }
 
-    public static HealthResult healthCheck()
+    public static HealthResult healthCheck(K8sAutoscaleConfiguration config)
```

Comment:
Parameters should be `final`:

```suggestion
    public static HealthResult healthCheck(final K8sAutoscaleConfiguration config)
```